MAUI/DataForm/Data-annotations.md

```diff
@@ -0,0 +1,284 @@
+</td>
+</tr>
+<tr>
+<td>
+{{'[Prompt]`Link`
+</td>
+<td>
+Specifies watermark text for the editor. Refer to `Link` for more details.
+</td>
+</tr>
+<tr>
+<td>
+{{'[Order] `Link`
+</td>
+<td>
+Specifies the order of field in the data form. Refer to `Link` for more details.
+</td>
+</tr>
+</table>
+
+## Validation attributes
+
+<table>
+<tr>
+<th>
+Property
+</th>
+<th>
+Details
+</th>
+</tr>
+<tr>
+<td>
+MinLength
+</td>
+<td>
+Specifies the required minimum length. Refer to `Link` for more details.
+</td>
+</tr>
+<tr>
+<td>
+MaxLength
+
+</td>
+<td>
+Specifies the required maximum length. Refer to `Link` for more details.
+</td>
+</tr>
+<tr>
+<td>
+Required
+
+</td>
+<td>
+Specifies the required data field value. Refer to `Link` for more details.
+</td>
+</tr>
+<tr>
+<td>
+Range
+</td>
+<td>
+Specifies the maximum and minimum values. Refer to `Link` for more details.
+</td>
+</tr>
+<tr>
+<td>
+StringLength
+</td>
+<td>
+Specifies the required string length. Refer to `Link` for more details.
+</td>
+</tr>
+</table>
+
+## Bindable attribute
+
+It specifies whether the field should be auto generated or not. Refer to `Link`  for more details.
+
+## Editable attribute
+
+It specifies whether the data field is editable or not.
+
+## ReadOnly attribute
+
+It specifies whether the data field is read only or not. Refer to `Link` for more details.
+
+## EnumDataType attribute
+
+It specifies enum type for the data field.
+
+## DataType attribute
+
+It specifies data type for the field.
+
+Supported data types are Text, MultilineText, Date, DateTime, Time, and Currency.
+
+Refer to `Link` for more details.
+
+## CustomDataType attribute
+
+The Percent data type is supported. Refer to `Link` for more details.
+
+## Custom attribute
+The data form supports the following custom attribute, and these attributes can be accessible using `Syncfusion.SfDataForm.XForms`.
+
+### DataFormDisplayOptions attribute
+
+<table>
+<tr>
+<th>
+Property
+</th>
+<th>
+Details
+</th>
+</tr>
+<tr>
+<td>
+{{'[RowSpan] `Link`
+</td>
+<td>
+Specifies the row span for the data form item. Refer to `Link` for more details.
+</td>
+</tr>
+<tr>
+<td>
+{{'[ColumnSpan] `Link`
+</td>
+<td>
+Specifies the column span for the data form item. Refer to `Link` for more details.
+</td>
+</tr>
+<tr>
+<td>
+{{'[ValidMessage] `Link`
+</td>
+<td>
+Specifies positive message to be shown when validation is passed. Refer to `Link` for more details.
+</td>
+</tr>
+<tr>
+<td>
+{{'[ShowLabel] `Link`
+</td>
+<td>
+Specifies whether the label should be visible or not. Refer to `Link` for more details.
+</td>
+</tr>
+</table>
+
+### DataFormValueConverter attribute
+
+<table>
+<tr>
+<th>
+Property
+</th>
+<th>
+Details
+</th>
+</tr>
+<tr>
+<td>
+{{'[DataFormValueConverterType] `Link`
+</td>
+<td>
+Specifies the Converter type which converts the original value in different format or as different value. Refer to here for more details.
+</td>
+</tr>
+</table>
+
+### DateRange attribute
```

Comment:
DataFormDateRange attribute

Reply:
DateRangeAttribute to DataFormDateRangeAttribute changed